### DISCUSSION TITLE
/report and /ticket Command

### DISCUSSION THREAD

**Discussion by cb12438** [Zrips/CMI#7728] [Feature Requests]
i think this would be useful to have seems like it would have it from the sheer amt of commands but would be nice to have it would alert staff members and maybe a gui for it to idk just would be nice :D

**Comment by mrfloris** [comment]
Issues is for bug reports, feature requests are to be posted here: https://github.com/Zrips/CMI/discussions

**Comment by CloudeLecaw** [comment]
Pls move it to discussions :)
